contributions/disa_import.py

```diff
@@ -78,19 +81,14 @@ def e(message: str) -> None:
             'name': NAME
         })
 
-    blank = next(rows)
-
-    for i in range(TYPE):
-        if blank[i].value:
-            e(_('Row %(row)s is not empty.') %
-              {'row': blank[i].row})
-            break
-
     for row in rows:
         if not any([row[i].value for i in range(TYPE)]):
-            break
-        if (not re.match(r'^\d+_', row[TITLE].value) and
-                row[TYPE].value != INFO):
+            continue
+        if (
+            not re.match(r'^\d+_', row[TITLE].value) and
+            row[TYPE].value != INFO and
+            not any([row[TITLE].value.startswith(x) for x in IGNORED_PREFIXES])
+        ):
```

Comment:
It is hard to see, whether the new condition is actually tested. No input new file was added or an old one changed. It could be an option, to add a private method/function `_check_title` or `_is_title_valid` and then have some parametrized tests to quickly unittest this aspect of the import.

Reply:
The test file ok_tools/test_data/valid.xlsx actually contains two contributions with the prefix 'Programmvorschau' and 'Trailer'. I don't know why this is not checked in with this PR. I thought a list might be better than private functions because it is easier to add more prefixes if the client decides to ignore more contribution with a specific prefix.

Reply:
I have nothing against the list, this can still be used in a private function. But as it is now, it is not possible to easily test those new prefixes without changing the xlsx all the time. It is also not visible in the code.

Reply:
That makes sense.

Reply:
[8caca0b](https://github.com/gocept/ok_tools/pull/84/commits/8caca0b473c7f3accbf05d8a5a3ba737ca4a9863)